winui/Avatar-view/customization.md

```diff
@@ -9,15 +9,15 @@ documentation: ug
 
 # Customization in WinUI AvatarView 
 
-The `AvatarView` control offers a range of customization options, including modifying its border, background, and more.
+The [SfAvatarView](https://www.syncfusion.com/winui-controls/avatar-view) control offers a range of customization options, including modifying its border, background, and more.
 
 ## Border
 
-Customize the `AvatarView` control border using the `BorderBrush` and `BorderThickness` properties.
+Customize the `AvatarView` control border using the [BorderBrush](https://learn.microsoft.com/en-us/windows/windows-app-sdk/api/winrt/microsoft.ui.xaml.controls.control.borderbrush?view=windows-app-sdk-1.2) and [BorderThickness](https://learn.microsoft.com/en-us/uwp/api/windows.ui.xaml.controls.control.borderthickness?view=winrt-22621) properties.
```

Comment:
@MadhumithanJayapathi use link for border brush and thickness like this
https://learn.microsoft.com/en-us/windows/windows-app-sdk/api/winrt/microsoft.ui.xaml.controls.control.borderthickness

https://learn.microsoft.com/en-us/windows/windows-app-sdk/api/winrt/microsoft.ui.xaml.controls.control.borderbrush

Reply:
@Manivannan-E Used the links given.